app/_data/docs_nav_mesh_2.0.x.yml

```
@@ -7,506 +7,1336 @@ items:
       - text: Version support policy
         url: /support-policy/
       - text: Stability
         url: /availability-stages/
-      - text: Kong Mesh requirements
-        url: /introduction/requirements/
-        src: /.repos/kuma/src/introduction/requirements/
       - text: Release notes
         url: /mesh/changelog
         generate: false
         absolute_url: true
-  - title: Get Started
+  - title: Geting Started
     icon: /assets/images/icons/documentation/icn-quickstart-color.svg
-    items:
-      - text: Overview
-        url: /get-started/
-        src: /.repos/kuma/src/get-started/
-      - text: Set up the Kubernetes demo app
-        url: /get-started/kubernetes-demo/
-        src: /.repos/kuma/src/get-started/kubernetes-demo/
-      - text: Set up the Universal demo app
-        url: /get-started/universal-demo/
-        src: /.repos/kuma/src/get-started/universal-demo/
-      - text: Setup a hybrid mesh app
-        url: /get-started/hybrid/
-        src: /.repos/kuma/src/get-started/hybrid/
-      - text: Get started with observability
-        url: /get-started/observability/
-        src: /.repos/kuma/src/get-started/observability/
-  - title: Kong Mesh in Production
+    url: /gettingstarted
+  - title: Install
     icon: /assets/images/icons/documentation/icn-deployment-color.svg
+    items:
+      - text: Kubernetes
+        url: /installation/kubernetes/
+        src: /.repos/kuma/app/docs/dev/installation/kubernetes/
+      - text: Helm
+        url: /installation/helm/
+        src: /.repos/kuma/app/docs/dev/installation/helm/
+        items:
+          - text: Argo CD
+            url: "/installation/helm/#argo-cd"
+            src: /.repos/kuma/app/docs/dev/installation/helm/#argo-cd
+      - text: OpenShift
+        url: /installation/openshift/
+        src: /.repos/kuma/app/docs/dev/installation/openshift/
+      - text: Docker
+        url: /installation/docker/
+        src: /.repos/kuma/app/docs/dev/installation/docker/
+      - text: Amazon ECS
+        url: /installation/ecs
+      - text: Amazon Linux
+        url: /installation/amazonlinux/
+        src: /.repos/kuma/app/docs/dev/installation/amazonlinux/
+      - text: Debian
+        url: /installation/redhat/
+        src: /.repos/kuma/app/docs/dev/installation/redhat/
+      - text: CentOS
+        url: /installation/centos/
+        src: /.repos/kuma/app/docs/dev/installation/centos/
+      - text: Debian
+        url: /installation/debian/
+        src: /.repos/kuma/app/docs/dev/installation/debian/
+      - text: Ubuntu
+        url: /installation/ubuntu/
+        src: /.repos/kuma/app/docs/dev/installation/ubuntu/
+      - text: macOS
+        url: /installation/macos/
+        src: /.repos/kuma/app/docs/dev/installation/macos/
+      - text: Windows
+        url: /installation/windows
+  - title: Deploy
+    icon: /assets/images/icons/documentation/icn-manager-color.svg
+    items:
+      - text: Explore Kuma with the Kubernetes demo app
```

```suggestion
      - text: Explore Kong Mesh with the Kubernetes demo app
```
Should these say Mesh?